omega/omega-transaction/src/main/java/org/apache/servicecomb/pack/omega/transaction/annotations/Compensable.java

```diff
@@ -39,15 +39,15 @@
 public @interface Compensable {
 
   /**
-   * The retires number of the compensable method.
```

Comment:
We need to update the document for it, it should be list in the upgrade guide.

Reply:
OK, I will add an upgrade-guide.md

Reply:
done